docs/userGuide/syntax/siteNavigationMenus.mbdf

```diff
@@ -14,13 +14,16 @@ Steps to add a siteNav:
 
 <code>
 * [:house: Home]({{ showBaseUrlText }}/index.html)<br>
-* Docs<br>
+---
+<br>* Docs<br>
 &nbsp;&nbsp;* [User Guide]({{ showBaseUrlText }}/ug.html)<br>
 &nbsp;&nbsp;* [Dev Guide]({{ showBaseUrlText }}/dg.html)<br>
-* [Search]({{ showBaseUrlText }}/search.html)<br>
+---
+<br>* [Search]({{ showBaseUrlText }}/search.html)<br>
 &nbsp;&nbsp;* [Google Search](https://www.google.com/)<br>
 &nbsp;&nbsp;* [YouTube Search](https://www.youtube.com/)<br>
-* [Contact]({{ showBaseUrlText }}/contact.html)
+---
+<br>* [Contact]({{ showBaseUrlText }}/contact.html)
 </code>
 </box>
```

Comment:
MarkBind now supports code blocks.

````
```{.no-line-numbers}
* [:house: Home]({{ baseUrl }}/index.html)
* Docs
  * [User Guide]({{ baseUrl }}/ug.html)
  * [Dev Guide]({{ baseUrl }}/dg.html)
* [Search]({{ baseUrl }}/search.html)
  * [Google Search](https://www.google.com/)
  * [YouTube Search](https://www.youtube.com/)
* [Contact]({{ baseUrl }}/contact.html)
```
````

Could be used instead. Much easier syntax for documentation.

Reply:
Want me to update it to using code blocks ?

Reply:
That would be nice. You could also change the other code blocks in this file into MarkBind code blocks, but that would expand the scope of your ticket by abit.

Reply:
Alright. Let me just change this to code block in this PR. Since this PR already affects this. For rest, I will do another PR

Reply:
Sounds good. Regarding that other PR, it would be best to wait after #1034 is merged. After that merge, the code blocks will have (almost) full functionality.

Reply:
Clicking on **Contact** shows Page Not Found. Any suggestion as what to do?

Reply:
@nbriannl can I get your review on this ?